src/AdminConsole/AuditLog/Loggers/AbstractAuditLoggerEfWriteStorage.cs

```diff
@@ -0,0 +1,30 @@
+using AdminConsole.Db;
+using Passwordless.AdminConsole.AuditLog.DTOs;
+using Passwordless.AdminConsole.AuditLog.Models;
+
+namespace Passwordless.AdminConsole.AuditLog.Loggers;
+
+public class AbstractAuditLoggerEfWriteStorage : IAuditLogger
+{
+    private readonly ConsoleDbContext _context;
+    private readonly List<OrganizationAuditEvent> _events = new();
+
+    protected AbstractAuditLoggerEfWriteStorage(ConsoleDbContext context)
```

Comment:
Should we just turn this one into AuditLoggerEfWriteSTorage and let Unauthneticated inherit from this? Seems one of our classes are empty.